src/components/action/usage/With-tooltip.md

```diff
@@ -0,0 +1,7 @@
+Renders a `calcite-action` that displays text along side an icon. A calcite-tooltip will be displayed on hover or focus.
```

Comment:
Typo: `along side` -> `alongside`

Reply:
According to the conventions, `calcite-tooltip` needs to have backticks: https://github.com/Esri/calcite-components/blob/master/conventions/Documentation.md#style-guide